packages/core/test/components/Autocomplete.test.tsx

```diff
@@ -164,6 +164,18 @@ describe('<Autocomplete />', () => {
 
       expect(wrapper.state('open')).toBe(true);
     });
+
+    it('calls loadItems if `loadItemsOnFocus` is `true`', () => {
```

Comment:
mind adding this test to Multicomplete too?

Reply:
doing this is pretty gross because 
- `loadItems` is not an instance method of `Multicomplete` so we can't spy on it
- we can't get a handle on the `autocomplete` instance because it's not the parent of `shallow()`
- spying on `onLoadOptions` requires a `timeout` because it's debounced 😭

Reply:
Currently we don't verify all of the other `Autocomplete` functionality in the `Multicomplete` tests, I feel like this isn't really necessary since the functionality lives in `Autocomplete` and we have tests there.

Reply:
oof - fair! wasn't sure and haven't checked this branch out locally. trying to cover all the bases that Multicomplete is 👍 too